code/src/App.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
 
+import Form from './components/Form' 
+
+
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <picture>
+        <source srcset="./images/galaxy.jpg" media="(max-width: 767px)" />
+        <source srcset="./images/moon-phases.jpg" />
+        <img class="hero-image" src="./images/moon-phases.jpg" />
+      </picture>
```

Comment:
I like the picture you chose! It looks cool. The text font also goes really well with your theme!